src/tasks/pollYoutube.js

```diff
@@ -0,0 +1,27 @@
+const { red, green } = require('colors/safe');
+const { schedule } = require('node-cron');
+const { fetchLatestYoutubeVideos } = require('../helpers/fetchData');
+const Video = require('../api/video/video.model');
+
+schedule('59 * * * *', async () => {
+  try {
+    let { date: lastDate } = await Video.findOne({}).sort({ date: -1 });
+    lastDate = lastDate.toISOString(); // Transforms date formart to the Youtube-API Standar.
+
+    const fetchedVideos = await fetchLatestYoutubeVideos({ publishedAfter: lastDate });
+    const now = new Date().toISOString();
```

Comment:
This should be declared right before it used or used inline as its usage is dependent on 2 if statements. It would also be more accurate the later it is declared.

Reply:
True. And if is not needed it won't be declared! 👌